app/src/renderer/components/governance/ModalVote.vue

```diff
@@ -7,6 +7,7 @@
     title="Vote"
     class="modal-vote"
     submission-error-prefix="Voting failed"
+    @close="$v.$reset()"
```

Comment:
do you need to add this to each modal if it's already on the action modal parent?